kolibri/plugins/learn/assets/src/learn_module.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var _ = require('lodash');
+var logging = require('loglevel');
+
+// This is aliased for your convenience!
+var KolibriModule = require('kolibri_module');
+
+var LearnModule = KolibriModule.extend({
+
+    start: function(plugin) {
+        // This is called on the kolibri_register event - make sure it is this plugin that has registered.
```

Comment:
Would be good to document the built-in events that are triggered by the asset loader. Also seems like this may be redundant with the `initialize` function -- is there a difference?